microhapulator/Snakefile

```diff
@@ -0,0 +1,369 @@
+# -------------------------------------------------------------------------------------------------
+# Copyright (c) 2022, DHS.
+#
+# This file is part of MicroHapulator (https://github.com/bioforensics/microhapulator) and is
+# licensed under the BSD license: see LICENSE.txt.
+#
+# This software was prepared for the Department of Homeland Security (DHS) by the Battelle National
+# Biodefense Institute, LLC (BNBI) as part of contract HSHQDC-15-C-00064 to manage and operate the
+# National Biodefense Analysis and Countermeasures Center (NBACC), a Federally Funded Research and
+# Development Center.
+# -------------------------------------------------------------------------------------------------
+
+from base64 import b64encode
+from datetime import datetime
+from glob import glob
+from jinja2 import Template
+import json
+from matplotlib import pyplot as plt
+import microhapulator
+from microhapulator.profile import TypingResult
+from os import symlink
+from pkg_resources import resource_filename
+import pandas as pd
```

Comment:
This might not be necessary, but to keep the alphabetical order in the python import statments, swapping `from pkg_resources` and `pandas` will help.